scripts/msi-test/run-msi-tests.sh

```
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+VM_HOST="192.168.13.87"
+VM_PORT=22
+VM_USER="jean"
+VM_PASS="cb6wist3"
+REMOTE_MSI="C:\\Users\\jean\\Downloads\\rocketchat-test.msi"
```

Shellcheck (0.11.0)

[warning] REMOTE_MSI appears unused. Verify use (or export if used externally).

(SC2034)